docs/maintainer-guide/pullrequests.md

```diff
@@ -25,13 +25,13 @@ Once the bot checks have been satisfied, you check the following:
     * Are there tests that verify the change? If not, please ask for them.
     * Is documentation needed for the change? If yes, please let the submitter know.
 1. Are there any automated testing errors? If yes, please ask the submitter to check on them.
-1. If you've reviewed the pull request and there are no outstanding issues, leave a comment "LGTM" to indicate your approval. If you're a reviewer and would like someone else to verify the change, comment "LGTM but would like someone else to verify."
+1. If you've reviewed the pull request and there are no outstanding issues, leave a comment "LGTM" to indicate your approval. If you're a TSC member and would like someone else to verify the change, comment "LGTM but would like someone else to verify."
```

Comment:
Good point. I think this was left over from before committers could merge anything.